lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -240,20 +242,22 @@ def check_initial_status(resources)
     end
     measure_method(:check_initial_status, "initial_status.duration")
 
-    def create_ejson_secrets(prune)
+    def secrets_from_ejson
       ejson = EjsonSecretProvisioner.new(
         namespace: @namespace,
         context: @context,
         template_dir: @template_dir,
         logger: @logger,
-        prune: prune,
+        statsd_tags: @namespace_tags
       )
-      return unless ejson.secret_changes_required?
 
-      @logger.phase_heading("Deploying kubernetes secrets from #{EjsonSecretProvisioner::EJSON_SECRETS_FILE}")
-      ejson.run
+      unless ejson.resources.empty?
```

Comment:
I'd suggest removing this logging entirely, since the inline resource list is showing which ones come from ejson